src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
 // write your code here
+const populationSpan = document.querySelectorAll('.population');
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+
+const populations = Array.from(populationSpan).map((span) => span.textContent);
+
+const parsedPopulations = populations
+  .map((pop) => {
+    const numberString = pop.split(',').join('');
+    const number = Number(numberString);
+
+    return isNaN(number) ? null : number;
+  })
+  .filter((num) => num !== null);
+
+const totalPopulation = parsedPopulations.reduce((sum, num) => sum + num, 0);
+const averagePopulation = totalPopulation / parsedPopulations.length;
```

Comment:
There is a potential division by zero error here. If `parsedPopulations` is empty, `parsedPopulations.length` will be zero, leading to a division by zero when calculating `averagePopulation`. Consider adding a check to handle this case.